crates/gitbutler-hunk-dependency/src/diff.rs

```diff
@@ -0,0 +1,76 @@
+use anyhow::{anyhow, Context};
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct Diff {
```

Comment:
Since i expect this to be an exported struct (saw it on the .add method) - we should probably name it something like DiffRange (since it does not include the diff itself)